src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs

```diff
@@ -68,7 +77,7 @@ public static void addCodeReloc(IntPtr thisHandle, void* handle)
             {
                 sb.Append("___SYMBOL");
             }
-            return (byte*)_this.GetPin(sb.UnderlyingArray);
+            return (byte*)_this.GetPin(AppendNullByte(sb.UnderlyingArray));
```

Comment:
You can append the null byte directly to the `Utf8StringBuilder` here. It will save a string copy.

Reply:
Thanks